src/Bridge/Symfony/Bundle/Resources/config/metadata.xml

```diff
@@ -116,24 +116,12 @@
 
         <!-- Cache -->
 
-        <service id="api_platform.metadata.resource.cache.array" class="Symfony\Component\Cache\Adapter\ArrayAdapter" public="false">
-            <argument>0</argument>
-            <argument>false</argument>
+        <service id="api_platform.metadata.resource.cache" parent="cache.system" public="false">
```

Comment:
Pertaining to symfony/symfony#18667, should our cache be considered a "system" or "app" cache? What are the semantics?